phpunit.xml.dist

```diff
@@ -5,35 +5,12 @@
 	convertErrorsToExceptions="true"
 	convertNoticesToExceptions="true"
 	convertWarningsToExceptions="true"
+	defaultTestSuite="default"
 	>
 	<testsuites>
-		<testsuite name="performance-lab">
+		<testsuite name="default">
```

Comment:
Could we utilize a root file as a configuration for all plugins, considering that all files have the same configuration? 🤔

Reply:
I did try it but can't find a way to achieve it with a single config 😞